WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -326,15 +327,16 @@ private extension InPersonPaymentsMenuViewController {
     }
 
     func configureSetUpTapToPayOnIPhone(cell: LeftImageTableViewCell) {
-        prepareForReuse(cell)
+        prepareForReuse(cell, accessibilityId: "set-up-tap-to-pay")
         cell.configure(image: UIImage(systemName: "wave.3.right.circle") ?? .creditCardIcon,
                        text: Localization.tapToPayOnIPhone)
     }
 
-    private func prepareForReuse(_ cell: UITableViewCell) {
+    private func prepareForReuse(_ cell: UITableViewCell, accessibilityId: String) {
```

Comment:
Nit: I use `Id` often and instinctively too, but generally end up changing it to `ID` which matches other convention in this codebase... might be worth doing here if you have a few minutes.

Reply:
Better to change it now rather than later, updated in https://github.com/woocommerce/woocommerce-ios/pull/9229/commits/b1be7c1d750abffa596c0bfc0583a0be2da0e14a